File: Q18/Q18.ts
// Count the number of occurrences of a character in a string.

function countOccurrences(str: string, char: string): number {
  let count = 0;
  for (let i = 0; i < str.length; i++) {
    if (str[i] === char) {
      count++;
    }
  }
  return count;
}

const str = "hello world";
const char = "l";
console.log(countOccurrences(str, char));

File: Q18/Q18.py
# Count the number of occurrences of a character in a string.

def count_occurrences(s, char):
    count = 0
    for c in s:
        if c == char:
            count += 1
    return count


s = "hello world"
char = "l"
print(count_occurrences(s, char)) 
